composer.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "drupal/media_avportal",
+    "description": "Media AV Portal adds European Audiovisual Services as a supported media provider.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": "^7.1",
+        "drupal/core": "^8.6",
+        "drupal/entity_browser": "~2.0",
+        "drupal/inline_entity_form": "~1.0"
```

Comment:
The entity browser and inline entity form are not required. For this PR or the other one.

Reply:
True. Removed in next commit.